config/proto/probe.proto

```diff
@@ -72,10 +62,15 @@ message HermesProbeDef {
     HOUR = 6;
   }
   
-  LatencyUnit latency_unit = 9;  // default is MICROSECOND
+  optional LatencyUnit latency_unit = 8;  // default is MICROSECOND
```

Comment:
Can you explain the purpose of this field?
I expect the cloudprober.metrics.Dist fields hold float values. Using seconds as the unit is the natural choice.

Reply:
The purpose of this field is to allow the user to define what latency unit to use for the latency metrics.
Yes, cloudprober.metrics.Dist fields do hold float64 values.

Seconds is a natural choice, but I did some testing on GCE, and the latency of an equivalent to ListBuckets() was 2 microseconds, which was a lot more representative to me as 2us, than 2e-6.

I'm open to changing this though, if you think that seconds would be more suitable, I can change it to that.

Reply:
2us is a human-readable rendering of the 2e-6 value.
In general: only convert something to human-readable form at the very last step (for example when rendering an HTML template or printing something in a CLI). The values should be transmitted/persisted in their rawest form.

Reply:
Okay, that's a very good point. Thank you. I'll stick with seconds until the last step of surfacing metrics.

Reply:
I think this field is still important though, as it decides the unit of the latency metric (which is included in the metric name).

Reply:
What do you mean by "decides the unit of the latency metric"? Is this a Cloudprober thing?

Reply:
I've removed the latency unit from the proto and added the measurement unit in the comments for the relevant proto fields.